### PR TITLE
Run rustfmt on src/test/codegen/ folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/test/codegen/drop.rs [hunk 1]
```diff
@@ -16,30 +16,28 @@
...
-// CHECK: call{{.*}}SomeUniqueName{{.*}}drop
-// CHECK: call{{.*}}SomeUniqueName{{.*}}drop
-// CHECK: call{{.*}}SomeUniqueName{{.*}}drop
-// CHECK: call{{.*}}SomeUniqueName{{.*}}drop
-// CHECK: call{{.*}}SomeUniqueName{{.*}}drop
-// CHECK: call{{.*}}SomeUniqueName{{.*}}drop
-// CHECK-NOT: call{{.*}}SomeUniqueName{{.*}}drop
-// The next line checks for the } that ends the function definition
-// CHECK-LABEL: {{^[}]}}
+    // Check that there are exactly 6 drop calls. The cleanups for the unwinding should be reused, so
```
Comment (eddyb) [inline src/test/codegen/drop.rs]: This line is too long.

**Comment by nikomatsakis** [comment]
@srinivasreddy seems like we still need to fix a line too long here

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to resubmit with a rebase!

**Pull request closed without merging**